.github/workflows/release.yml

```diff
@@ -37,7 +37,6 @@ jobs:
         with:
           version: ${{ steps.get_version.outputs.version }}
           git_tag_prefix: v
-          
       - name: Create new tag
         run: |
           if [ -z "${{ steps.get_version.outputs.version }}" ]; then
```

Comment:
It seems need to update too about using the optimizer version and based rust version in `Build development contracts` and `Check development contracts`.
```

      - name: Build development contracts
        run: |
          echo "Building all contracts under ./contracts"
          docker run --volumes-from with_code cosmwasm/rust-optimizer:0.11.0 ./contracts/*/
      - name: Check development contracts
        working-directory: ./
        run: |
          echo "Checking all contracts under ./artifacts"
          docker run --volumes-from with_code rust:1.51.0 /bin/bash -e -c 'cd ./code/packages/vm; ./examples/check_contract.sh ../../artifacts/*.wasm'
          docker cp with_code:/code/artifacts .
```

Reply:
Upgraded to 0.12.1.